test/multivariate/dirichlet.jl

```diff
@@ -39,6 +39,7 @@ rng = MersenneTwister(123)
         @test iszero(pdf(d, [0, 0, 1]))
         @test pdf(d, [0.2, 0.3, 0.5]) ≈ 3.6
         @test pdf(d, [0.4, 0.5, 0.1]) ≈ 2.4
+        @test typeof(pdf(d, missing)) == Missing
```

Comment:
Simpler:
```suggestion
        @test pdf(d, missing)) === missing
```
IMO it would also be better to not add this check to all kinds of different files but to the generic test interfaces in e.g. `test/testutils.jl`.